### PR TITLE
PRIME-1534 add signature block to pdf generate.

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

File: prime-dotnet-webapi/Views/Agreements/CommunityPharmacyOrganizationAgreement.cshtml [hunk 1]
```diff
@@ -586,7 +586,12 @@
   </li>
 </ol>
 
-<p class="text-center">
+@if (@Model.WithSignature)
```
Comment (AnaisHebert) [inline prime-dotnet-webapi/Views/Agreements/CommunityPharmacyOrganizationAgreement.cshtml:589]: Just want to confirm that they want the singature block before the Schedule A section?

Comment (mtpultz) [reply prime-dotnet-webapi/Views/Agreements/CommunityPharmacyOrganizationAgreement.cshtml:589]: Yep

**Review by AnaisHebert — APPROVED** [review]
✨✨✨

File: prime-dotnet-webapi/Controllers/OrganizationsController.cs [hunk 2]
```diff
@@ -308,19 +308,25 @@ public async Task<ActionResult<Agreement>> GetOrganizationAgreement(int organiza
...
         [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
         [ProducesResponseType(typeof(ApiResultResponse<string>), StatusCodes.Status200OK)]
-        public async Task<ActionResult<string>> GetSignableOrganizationAgreement(int organizationId, int agreementId)
+        public async Task<ActionResult<string>> GetSignableOrganizationAgreement(int organizationId, [FromQuery] AgreementType agreementType)
         {
-            var pdf = await _agreementService.GetSignableOrgAgreementAsync(organizationId, agreementId);
+            var allowedAgreementTypes = new[] { AgreementType.CommunityPracticeOrgAgreement, AgreementType.CommunityPharmacyOrgAgreement };
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Controllers/OrganizationsController.cs:320]: We have an Extension method for IsEnrolleeAgreement. I would prefer to see this as either !IsEnrolleeAgreement or as a new Extension method IsOrganizationAgreement

**Review by james-hollinger — APPROVED** [review]
L "LGTM" TM

**Pull request merged**